plugins/in_windows_exporter_metrics/we_wmi.c

```diff
@@ -150,6 +150,32 @@ static int wmi_update_counters(struct wmi_query_spec *spec, uint64_t timestamp,
     return 0;
 }
 
+static char *convert_prop_to_str(VARIANT *prop, int handle_null)
+{
+    char *strlabel = NULL;
+    char *newstr = NULL;
+
+    if (handle_null == FLB_TRUE && prop->vt == VT_NULL) {
+        newstr = strdup("");
+        if (newstr == NULL) {
+            return NULL;
+        }
+    }
+    else {
+        VariantChangeType(prop, prop, 0, VT_BSTR);
+        strlabel = we_convert_wstr(prop->bstrVal, CP_UTF8);
+        if (strlabel == NULL) {
+            return NULL;
+        }
+        newstr = strdup(strlabel);
+        if (newstr == NULL) {
+            return NULL;
```

Comment:
You need to release `strlabel` here as well, otherwise you will leak it if the previous strdup fails.